src/sig_stfl/sig_stfl.h

```diff
@@ -271,26 +271,29 @@ typedef struct OQS_SIG_STFL_SECRET_KEY {
 	/* file storage handle */
 	void *context;
 
+	/* Secret key lock status: 0: key is locked, otherwise key is unlocked */
+	bool is_locked;
+
 	/**
 	 * Secret Key retrieval Function
 	 *
 	 * @param[in] sk The secret key represented as OQS_SIG_STFL_SECRET_KEY object
 	 * @param[out] sk_len length of private key as a byte stream
 	 * @param[out] sk_buf_ptr pointer to private key data as a byte stream
 	 * @returns length of key material data available
-	 * Caller deletes the buffer if memory was allocated.
+	 * Caller is responsible to **unallocate** the pointer to buffer `sk_buf_ptr`.
 	 */
-	OQS_STATUS (*serialize_key)(const OQS_SIG_STFL_SECRET_KEY *sk, size_t *sk_len, uint8_t **sk_buf_ptr);
```

Comment:
Any reason to not keep this a 'const'?

Reply:
Yeah @ashman-p , because we modify the `sk->is_locked` variable, so I have to remove the `const`. what do you think?